shopelectro/tests/helpers.py

```diff
@@ -1,11 +1,15 @@
 import base64
 from contextlib import contextmanager
+import re
```

Comment:
`import` group should be before `from ... import ...`, isn't it?

Reply:
@artemiy312 , i was suprised by myself too, but at cryptobot project coala checked only lib names, without keywords. pep8 don't contain concrete recomendations about import/from order too.

I'll check linter again
TODO

Reply:
>pep8 don't contain concrete recomendations about import/from order too.

as I remember we decided to stick such ordering rules:
The first block is std-lib
The second is third-party
The third is local packages
The fourth is refarm-site packages

Each of group is ordered by import type and alphabetic. `import ...` blocks are higher than `from ... import ...`

Reply:
y, i'll fix it because of current rules
TODO